example/lib/location_indicator_example_page.dart

```diff
@@ -71,16 +72,21 @@ class _LocationIndicatorExamplePageState
                 _controller = controller;
                 _requestLocationPermission();
                 _configureLocationDisplay(Colors.blue);
+
+                _panUpdateSubscription?.cancel();
+                _panUpdateSubscription = controller
+                    .centerPosition()
+                    .debounceTime(const Duration(milliseconds: 50))
```

Comment:
rxdart debounce makes it impossible to test with fakeasync https://github.com/ReactiveX/rxdart/issues/395
Either use another debounce package or remove the debounce? This should be a user decision

Reply:
I am now using a `Timer` instead of `debounce`.